lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -135,12 +136,16 @@ def join_bulk_responses(bulk_responses)
     def bulk_send(body_stream)
       params = http_compression ? {:headers => {"Content-Encoding" => "gzip"}} : {}
       # Discard the URL 
+      @metric.increment(:bulk_requests)
+      @metric.increment(:concurrent_bulk_requests)
```

Comment:
@allenmchan any feedback here regarding the usage of concurrent bulk requests?

FYI we're discussing the overall ES output stats here: https://github.com/elastic/logstash/issues/6831#issuecomment-295691299